packages/shared/src/utils.ts

```diff
@@ -44,22 +44,22 @@ const sortedReplacer = (_key: string, value: unknown) => {
   return value;
 };
 
-const u8ToBase64Replacer = (_key: string, value: unknown) => {
+const u8ToHexReplacer = (_key: string, value: unknown) => {
   if (value instanceof Uint8Array) {
-    return `base64:${toBase64(value)}`;
+    return `0x${toHexString(value)}`;
   }
   return value;
 };
 
 const sortedSerializingReplacer = (_key: string, value: unknown): unknown => {
-  const serializedValue = u8ToBase64Replacer(_key, value);
+  const serializedValue = u8ToHexReplacer(_key, value);
```

Comment:
Would this also implicitly apply to `paremeters` sent in the condition i.e. if a `parameter` value was in bytes...?

Reply:
Yes, this is what all JSON serialization methods are using in `nucypher-ts`.

Reply:
👍